src/test/java/org/openhab/binding/jrule/internal/triggers/itemchange/JRuleGroupItemChangeTest.java

```diff
@@ -49,74 +50,110 @@ public void initTestClass() throws ItemNotFoundException {
+//            Mockito.when(stringItem.getGroupNames()).thenReturn(List.of(JRuleGroupItemChangeRules.GROUP_ITEM));
+//            return stringItem;
+//        });
+//
+//        JRuleGroupItemChangeRules rule = initRule(JRuleGroupItemChangeRules.class);
+//        // Only last event should trigger rule method
+//        fireEvents(List.of(itemChangeEvent(OTHER_ITEM, "2", "1"), itemChangeEvent(MEMBER_ITEM, "2", "1")));
+//        verify(rule, times(1)).groupItemChange(Mockito.any(JRuleEvent.class));
+//    }
 
     @Test
-    public void testItemChange_to() throws ItemNotFoundException {
+    public void testItemChangeJustItems() throws ItemNotFoundException {
         Mockito.when(itemRegistry.getItem(MEMBER_ITEM)).then((Answer<Item>) invocationOnMock -> {
             StringItem stringItem = Mockito.mock(StringItem.class);
             Mockito.when(stringItem.getName()).thenReturn(invocationOnMock.getArgument(0));
-            Mockito.when(stringItem.getGroupNames()).thenReturn(List.of(JRuleGroupItemChangeRules.GROUP_ITEM_TO));
+            Mockito.when(stringItem.getGroupNames())
+                    .thenReturn(List.of(JRuleGroupItemChangeRules.GROUP_ITEM_JUST_ITEMS));
             return stringItem;
         });
+        Mockito.when(itemRegistry.getItem(JRuleGroupItemChangeRules.GROUP_ITEM_JUST_ITEMS))
+                .thenAnswer(invocationOnMock -> new StringItem("Any"));
 
         JRuleGroupItemChangeRules rule = initRule(JRuleGroupItemChangeRules.class);
-        // Only last event should trigger rule method
-        fireEvents(List.of(itemChangeEvent(MEMBER_ITEM, "1", "2"), itemChangeEvent(MEMBER_ITEM, "2", "1")));
-        verify(rule, times(1)).groupItemChangeTo(Mockito.any(JRuleEvent.class));
+        fireEvents(List.of(itemChangeEvent(MEMBER_ITEM, "2", "1")));
+//        verify(rule, times(0)).groupItemChangeJustGroups(Mockito.any(JRuleEvent.class));
+        verify(rule, times(1)).groupItemChangeJustItems(Mockito.any(JRuleEvent.class));
     }
 
-    @Test
-    public void testItemChange_from_to() throws ItemNotFoundException {
-        Mockito.when(itemRegistry.getItem(MEMBER_ITEM)).then((Answer<Item>) invocationOnMock -> {
-            StringItem stringItem = Mockito.mock(StringItem.class);
-            Mockito.when(stringItem.getName()).thenReturn(invocationOnMock.getArgument(0));
-            Mockito.when(stringItem.getGroupNames()).thenReturn(List.of(JRuleGroupItemChangeRules.GROUP_ITEM_FROM_TO));
-            return stringItem;
-        });
-
-        JRuleGroupItemChangeRules rule = initRule(JRuleGroupItemChangeRules.class);
-        // Only last event should trigger rule method
-        fireEvents(List.of(itemChangeEvent(MEMBER_ITEM, "2", "1"), itemChangeEvent(MEMBER_ITEM, "3", "2"),
-                itemChangeEvent(MEMBER_ITEM, "1", "2")));
-        verify(rule, times(1)).groupItemChangeFromTo(Mockito.any(JRuleEvent.class));
-    }
+//    @Test
```

Comment:
A lot of commented out tests - should these be enabled again?

Reply:
uncommented